pkg/nsx/services/securitypolicy/parse.go

```diff
@@ -36,11 +36,3 @@ func getRuleDirection(rule *v1alpha1.SecurityPolicyRule) (string, error) {
 	}
 	return "", errors.New("invalid rule direction")
 }
-
-func getCluster(service *SecurityPolicyService) string {
-	return service.NSXConfig.Cluster
-}
-
-func getDomain(service *SecurityPolicyService) string {
```

Comment:
we had better to keep getDomain function or put it as common function, although it's short function, but it's more meaningful compared to use "service.NSXConfig.Cluster".